discord-scripts/invite-management.ts

```diff
@@ -0,0 +1,134 @@
+import { Robot } from "hubot"
+import { Client, TextChannel, VoiceChannel, StageChannel } from "discord.js"
+
+async function manageInvite(
+  discordClient: Client,
+  guildId: string,
+  channelId: string,
+  action: "create" | "delete",
```

Comment:
Let's not get ahead of our skates here—if we're not supporting `delete` yet (a correct decision), let's just call this `createInvite` and skip the action stuff.

Most likely the right way to approach if we do add deleting will be to add a `deleteInvite` and extract the common lookup stuff to a common helper.